File: Exemplos M10/App_IntentDigitalHUB_Kotlin/Elgin_IntentDigitalHUB_Kotlin-font/app/src/main/java/com/elgin/intent_digital_hub/IntentDigitalHubService/TERMICA/Commands/AvancaPapel.kt
package com.elgin.intent_digital_hub.IntentDigitalHubService.TERMICA.Commands

import com.elgin.intent_digital_hub.IntentDigitalHubService.TERMICA.TermicaCommand

class AvancaPapel(private val linhas: Int) : TermicaCommand("AvancaPapel") {
    override fun functionParameters(): String {
        return "\"linhas\"" + ":" + linhas
    }
}

File: Exemplos SmartPOS/App_IntentDigitalHUB_Flutter/Elgin_IntentDigitalHUB_SmartPOS_Flutter-font/lib/IntentDigitalHubService/TERMICA/Commands/avanca_papel.dart
import 'package:flutter_smartpos/IntentDigitalHubService/TERMICA/termica_command.dart';

class AvancaPapel extends TermicaCommand {
  final int _linhas;

  AvancaPapel(this._linhas) : super('AvancaPapel');

  @override
  get functionParametersJson => {
        'linhas': _linhas,
      };
}
